blog/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path, include
+from .views import PostListView, PostDetailView
+
+urlpatterns = [
+    path("", PostListView.as_view(), name="index"),
+    path("<int:pk>/", PostDetailView.as_view(), name="post-detail"),
```

Comment:
Accordint to task requirements your path should contains `posts`
`Add a template for this page and url with a path posts/pk/ and name post-detail.`